doc/releases/changelog-0.37.0.md

```diff
@@ -6,57 +6,171 @@
 
 <h4>Execute faster with Default Tensor 🔗</h4>
 
-* The `default.tensor` device now supports the `tn` method to simulate quantum circuits using exact tensor networks.
+* A new `default.tensor` device is now available for performing
+  [tensor network](https://en.wikipedia.org/wiki/Tensor_network) and
+  [matrix product state](https://en.wikipedia.org/wiki/Matrix_product_state) simulations
+  of quantum circuits using the
+  [quimb backend](https://quimb.readthedocs.io/en/latest/).
+  [(#5699)](https://github.com/PennyLaneAI/pennylane/pull/5699)
+  [(#5744)](https://github.com/PennyLaneAI/pennylane/pull/5744)
   [(#5786)](https://github.com/PennyLaneAI/pennylane/pull/5786)
+  [(#5795)](https://github.com/PennyLaneAI/pennylane/pull/5795)
 
-* The `default.tensor` device is introduced to perform tensor network simulations of quantum circuits using the `mps` (Matrix Product State) method.
-  [(#5699)](https://github.com/PennyLaneAI/pennylane/pull/5699)
+  Either method can be selected when instantiating the `default.tensor` device by setting the
+  `method` keyword argument to `"tn"` (tensor network) or `"mps"` (matrix product state). This
+  device can simulate a large number of qubits with sensible approximations. Take this example that
+  can simulate 1000 qubits in a few seconds!
```

Comment:
```suggestion
  `method` keyword argument to `"tn"` (tensor network) or `"mps"` (matrix product state). This
  device can simulate a large number of qubits with low-to-intermediate entanglement. Take this example that
  can simulate 1000 qubits with a shallow circuit in a few seconds!
```

The full contraction method (in its current form) is always exact and there is no approximation. Both methods have different strengths, but it would be correct to generally say for both that they are good for simulating low-to-intermediately entangled circuits. "Low-to-intermediate" is a bit of a mouthful an vague, could be replaced by something else in case you dont like it :)

Reply:
This is always a tough thing to describe without going into too much detail, so the nuance I left out with what's currently written is probably confusing you — totally understandable 😅. The point I'm making is akin to saying "trotterization is efficient with sensible approximations". The implementation of the method under the hood in PennyLane is exact, but certainly the trotterization technique is approximating something. 

So, with MPS, the implementation of the contraction method is exact, and anything under the sun can be represented by an MPS... _with a big enough bond dimension_, and that's what I'm talking about when I say "approximations". If a crappy bond dimension is chosen, the result is an approximation of the exact result. 

(_takes breath_)

What's probably better to say here is this (leave even more specifics out 😆, and let the user discover the wonderful world of matrix product states)

```suggestion
  `method` keyword argument to `"tn"` (tensor network) or `"mps"` (matrix product state). This
  device can efficiently simulate a large number of qubits. Take this example that
  can simulate 1000 qubits in a few seconds!
```

The reason I don't love "low-to-intermediate entanglement" here is because you can have lots of entanglement so long as we have area-law scaling. 

TLDR; let's just be super vague 😆

Reply:
I get your point about the MPS method, just want to clarify that the full contraction method (`method="tn"`) really is exact-exact and there is (currently) no way to introduce approximations.

Reply:
Oooooooooh okay okay... apologies 😆 I over explained the crap out of my perspective then lol. In that case, I still prefer my suggestion above. You good with that? (emphasis on the "can" in "can efficiently simulate ...")